bundles/org.openhab.core/src/test/java/org/openhab/core/util/ColorUtilTest.java

```diff
@@ -389,7 +389,7 @@ public Stream<? extends Arguments> provideArguments(@Nullable ExtensionContext c
      */
     private static Stream<Arguments> allHSB() {
         List<Arguments> result = new ArrayList<>();
-        final double step = 5.0;
+        final double step = 17.0;
```

Comment:
Do you want to hit the upper boundaries 100 and 360? Then a factor of 100 and 360 would be a better choice.
With step size 17, the loop will end at 85 and 357, respectively. 10 or 20 might be a better choice.

Reply:
See above.